frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -31,18 +31,23 @@ function FileManagerMenu:init()
         -- items in top menu
         filemanager_settings = {
             icon = "appbar.filebrowser",
+            text = _("Filebrowser"),
         },
         setting = {
             icon = "appbar.settings",
+            text = _("Settings"),
```

Comment:
Now that you have set the icon in the ConfirmBox, do we need to still have these (translated) text here.
And do we also need it in the ConfirmBox (this text would be shown nowhere else, so it relates to nothing, except giving a name to the icon on its left in the ConfirmBox.